### PR TITLE
Move feedback into footer, fixes part of #15

### PR DESCRIPTION
**Pull request by DanielDugan**
Moved Feedback button into footer.
Fixed footer in .scss
Created webhook for "Updated date"

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -33,16 +33,16 @@ ARG LOCALHOST_DEV
 
 WORKDIR /usr/src/app/student_explorer/dependencies/
 
-ENV ORACLE_HOME /usr/lib/oracle/18.3/client64	
-ENV LD_LIBRARY_PATH /usr/lib/oracle/18.3/client64/lib	
+ENV ORACLE_HOME /usr/lib/oracle/18.3/client64
```
[the remark this answers is not in the document]
Comment (jonespm) [reply Dockerfile]: Yeah I don't know why there was so much whitespace there? Maybe something in the @DanielDugan 's IDE cleaning it up?

[the remark this answers is not in the document]
Comment (DanielDugan) [reply Dockerfile]: Investigating

[the remark this answers is not in the document]
Comment (jonespm) [reply Dockerfile]: I think it's okay for this commit, just might want to see if your IDE is reformatting code as it shouldn't be making whitespace only changes.

[the remark this answers is not in the document]
Comment (jonespm) [reply Dockerfile]: I can merge this if you were going to have a new branch for the fix of updating the footer.

[the remark this answers is not in the document]
Comment (DanielDugan) [reply Dockerfile]: Not seeing the extra whitespaces on this side. Using Atom. testing now

[the remark this answers is not in the document]
Comment (DanielDugan) [reply Dockerfile]: okie dokie. Looks like using hotkey save "cmd+s" generates a newline in atom as wel. I am not sure why.

**Pull request merged**